kolibri/core/auth/migrations/0014_auto_20190815_1421.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.22 on 2019-08-15 21:21
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import kolibri.core.auth.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("kolibriauth", "0013_auto_20180917_1213")]
+
+    operations = [
+        migrations.AddField(
+            model_name="facilityuser",
+            name="birth_year",
+            field=models.CharField(
+                blank=True,
+                default="",
+                max_length=16,
+                validators=[kolibri.core.auth.models.validate_birth_year],
+            ),
+        ),
+        migrations.AddField(
+            model_name="facilityuser",
+            name="gender",
+            field=models.CharField(
+                blank=True,
+                choices=[
+                    ("MALE", "Male"),
+                    ("FEMALE", "Female"),
+                    ("NOT_SPECIFIED", "Not specified"),
+                    ("DEFERRED", "Defers for later"),
+                ],
```

Comment:
Some check constraints for gender and age could be worthwhile.

Reply:
In auth.models.FacilityUser, I think I get the checks for free by setting the `choices` kwarg. I also wrote a validator for the `birth_year` char field